holoviews/core/spaces.py

```diff
@@ -327,14 +327,15 @@ def sample(self, samples=[], bounds=None, **sample_values):
                 ysamples = [(ly+uy)/2.0 for ly,uy in zip(yedges[:-1], yedges[1:])]
 
                 Y,X = np.meshgrid(ysamples, xsamples)
-                linsamples = zip(X.flat, Y.flat)
+                linsamples = list(zip(X.flat, Y.flat))
```

Comment:
Guess this isn't tested? Wouldn't there have been Python 3 issues if it had been?